### DISCUSSION TITLE
2024-01-13-open

### DISCUSSION THREAD

**Discussion by giscus** [tw93/tw93.github.io#167] [General]
# 我的开源成长之旅 - Tw93

Hi，我是 Tw93，是一个前端工程师，会一点 Swift / Rust / Python，喜欢折腾各种新技术。Thought在上海微软 Github Universe 2023 的一个分享，主要是讲我从大学开源到现在 9 年做的一些有趣的事情，希望可以给热爱开源的朋友一些输入。

https://tw93.fun/2024-01-12/open.html

**Comment by nicekate** [comment]
非常好的分享，体验过你的多款产品，认可你的开源理念，祝福你越来越好

**Reply by tw93** [reply]
谢谢喜欢，假如有觉得更好的建议，也很欢迎直接提

**Comment by jeasonnow** [comment]
能从中得到快乐，扩展知识，结交志同道合的朋友，这样的开源真的很棒。🎉

**Reply by PuddingJoker** [reply]
作者提到的那个大佬嘛

**Comment by jalenzz** [comment]
Hi，请问 代码为什么要开源 那一页的动图是如何生成的

**Reply by tw93** [reply]
可以参考我之前的这篇文章
https://tw93.fun/2022-07-09/code.html

**Comment by stevobm** [comment]
看了你的作品和文章，确实是个开源达人。公司大部分代码都商业化上云了，能够开源出来确实比较珍贵。

**Comment by davidchen27** [comment]
真的很喜欢妙言的界面跟功能呀, 可惜没有windows版本😭   请问后面有没有考虑搬到win平台呀

**Reply by tw93** [reply]
可能会有计划用其他语言来实现，现在是swift版本，不太好迁移，除非可能用rust来搞搞
